files/build_templates/sonic_debian_extension.j2

```diff
@@ -78,6 +78,10 @@ fi
 # Update apt's snapshot of its repos
 sudo LANG=C DEBIAN_FRONTEND=noninteractive chroot $FILESYSTEM_ROOT apt-get update
 
+# Install efitools to support secure upgrade
+sudo LANG=C DEBIAN_FRONTEND=noninteractive chroot $FILESYSTEM_ROOT apt-get -y install efitools
```

Comment:
>efitools [](http://example.com/codeflow?start=86&length=8)

We should only install these 2 packages in UEFI platforms?

Reply:
we are installing the `efitools` always, and the following just when the feature is enabled:
shim-unsigned
grub-efi
code can be found in `build_debian.sh`.

Reply:
you comment is not relevant to my comment.

Reply:
pls can you elaborate your comment?

Reply:
I think now Im understand your question, efitools is only for UEFI platform, so we can add a condition that checking if the platform support UEFI and if yes install, other not.
I will add this condition, thanks